src/storage/remote/write.c

```diff
@@ -215,6 +192,7 @@ storageWriteRemoteNew(
         {
             .storage = storage,
             .client = client,
+            .session = protocolClientSessionNewP(client, PROTOCOL_COMMAND_STORAGE_WRITE, .async = true),
```

Comment:
Before, there was essentially one session that everyone shared so there was no need to explicitly request a session. In most cases you still do not need to explicitly request a session since protocolClientRequestP() does it for you (and that's how most requests are done). But for read/write we need an explicit session so this is where it is created.